drivers/w1/Kconfig.ds2484

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2022, Caspar Friedrich <c.s.w.friedrich@gmail.com>
```

Comment:
See https://github.com/zephyrproject-rtos/zephyr/pull/48166 on new style for how we are doing Kconfig/DT stuff based on autogenerated Kconfig.dts files (look at <BUILD>/Kconfig/Kconfig.dts)

Reply:
Fixed